Monitoring/metadata/V3/Notification.php

```diff
@@ -15,46 +15,48 @@ public static function initOnce() {
           return;
         }
         \GPBMetadata\Google\Api\Label::initOnce();
+        \GPBMetadata\Google\Api\LaunchStage::initOnce();
```

Comment:
We're going to need to get this generated in: https://github.com/googleapis/common-protos-php